### DISCUSSION TITLE
Do you need to add the type of TAPP?

### DISCUSSION THREAD

**Discussion by DISTANCE122** [tearust/teaproject#81] [Ideas]
I personally feel it is necessary to add more types of TAPP, such as Chinese social media APP Weibo Bili and so on. There are also many Chinese friends in the game who don’t have fans on Youtube, but there are probably tens of thousands of followers in places like Weibo. I’m from China, and I can promote the tea project through Weibo

**Comment by kevingzhang** [comment]
There are two biggest challenges here.
First, crypto mining is illegal in China. Any centralized application like Weibo or Wechat is controlled by the central authority. We need to have the API to weibo to get the social counts. No doubt, the API will be revoked by the central authority if someone report we are using the API for crypto mining related usage.
Second, the firewall actually blocks many API call from outside of mainland China. 

In epoch6, we have a new TEA Application called Tea Party (Tpty). This is a replacement of Weibo or Wechat. Although it is a Proof of Concept right now with only very limited basic feature. But it is decentralized. It cannot be controlled by any central authority. I would like users from China, Iran or Afghanistan to use this kind of apps instead. Who use weibo or wechat can share links to this Tea Party.
